apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -21,16 +21,18 @@ export const removeStudent = (studentId) => ({ type: REMOVE_STUDENT, studentId }
 export const setSecretImage = (studentId, image) => ({ type: SET_SECRET_IMAGE, studentId, image });
 export const setSecretWords = (studentId, words) => ({ type: SET_SECRET_WORDS, studentId, words });
 export const editStudent = (studentId, studentData) => ({ type: EDIT_STUDENT, studentId, studentData });
+export const startSavingStudent = (studentId) => ({ type: START_SAVING_STUDENT, studentId });
+export const saveStudentSuccess = (studentId) => ({ type: SAVE_STUDENT_SUCCESS, studentId });
 
 export const saveStudent = (studentId) => {
   return (dispatch, getState) => {
     const state = getState().manageStudents;
-    dispatch({ type: START_SAVING_STUDENT, studentId });
+    dispatch(startSavingStudent(studentId));
     updateStudentOnServer(state.editingData[studentId], (error, data) => {
       if (error) {
         console.error(error);
       }
-      dispatch({ type: SAVE_STUDENT_SUCCESS, studentId });
+      dispatch(saveStudentSuccess(studentId));
```

Comment:
nit/optional: `saveStudentComplete` might be more accurate, since we are doing this regardless. 

Also, what happens if the save fails?

Reply:
Right now it just prints a console error, but I think we could implement something nicer as a next step. 
`SAVE_STUDENT_SUCCESS` AND `SAVE_STUDENT_FAILURE` were the actions I was thinking.

Reply:
👍 those sound great